fabric-loot-table-api-v2/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,26 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-loot-table-api-v2",
+  "name": "Fabric Loot Table API (v2)",
+  "version": "${version}",
+  "environment": "*",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-loot-table-api-v2/icon.png",
+  "contact": {
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.4.0",
+    "fabric-api-base": "*"
+  },
+  "description": "Hooks for manipulating loot tables.",
+  "mixins": [
+    "fabric-loot-table-api-v2.mixins.json"
+  ]
```

Comment:
This module now needs to define a module lifecycle:
For the new module you would add the following to the fabric.mod.json
```json
  "custom": {
    "fabric-api:module-lifecycle": "stable"
  }
```